new/PRS13297Zamalakot.xml

```diff
@@ -0,0 +1,68 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13297Zamalakot" xml:lang="en" type="pers">
```

Comment:
Did you reactivate this branch? It was already merged. The first file (PRS13297Zamalakot) is already in the master branch and in the app:
https://betamasaheft.eu/persons/PRS13297Zamalakot/main

Strange is hoewever, that there is no "title" in the file. See also here, no "name" for him in the result list: https://betamasaheft.eu/persons/list?mainname=Zamalakot&date-range=0%2C2000&cp=

In any case, the first file probably needs to be deleted, and changes on this  record (PRS13297Zamalakot) should be done in a new branch. Is this correct? @thea-m  @PietroLiuzzo

Reply:
https://github.com/BetaMasaheft/Documentation/issues/1603